src/main/java/org/jvnet/hudson/test/HudsonTestCase.java

```diff
@@ -685,7 +685,7 @@ public PretendSlave createPretendSlave(FakeLauncher faker) throws Exception {
     public ComputerLauncher createComputerLauncher(EnvVars env) throws URISyntaxException, IOException {
         int sz = jenkins.getNodes().size();
         return new SimpleCommandLauncher(
-                String.format("\"%s/bin/java\" %s %s -jar \"%s\"",
+                String.format("\"%s/bin/java\" %s %s Xmx512M -jar \"%s\"",
```

Comment:
doesn't this need a `-` ?
```suggestion
                String.format("\"%s/bin/java\" %s %s -Xmx512M -jar \"%s\"",
```

Reply:
yes it does - good catch :)